internal/controller/controllers/controller_event_wrapper.go

```diff
@@ -27,6 +28,11 @@ func NewControllerEventsWrapper(crdEventsHandler CRDEventsHandler, events *event
 
 func (c *controllerEventsWrapper) AddEvent(ctx context.Context, clusterID strfmt.UUID, hostID *strfmt.UUID, severity string, msg string, eventTime time.Time, props ...interface{}) {
 	c.events.AddEvent(ctx, clusterID, hostID, severity, msg, eventTime, props)
+	if severity == models.EventSeverityInternal {
+		return
```

Comment:
Do we really want to drop it? Shouldn't it be available according to RBAC?

Reply:
The controller is only interested in cluster/host update events. There are too many events getting to controller as it is, but internal events certainly should not reach it